cypress.config.ts

```diff
@@ -2,8 +2,16 @@ const { defineConfig } = require("cypress");
 
 module.exports = defineConfig({
   e2e: {
+    video: false,
+    screenshotOnRunFailure: false,
+    chromeWebSecurity: false,
     setupNodeEvents(on, config) {
-      // implement node event listeners here
-    },
+    // e2e testing node events setup code
+    config.defaultCommandTimeout= 10000;
+    config.responseTimeout= 20000;
+    config.pageLoadTimeout= 60000;
+    return config;
   },
```

Comment:
fix the indentation for these lines, as they are part of the setupNodeEvents, you should add one indent more.

Reply:
Solved